### DISCUSSION TITLE
SRT format gopro 11

### DISCUSSION THREAD

**Discussion by Blackwolf8202** [gopro/labs#423] [Feature Requests]
Maybe add SRT format in GoPro 11? so you can have the option to stream with it with and not only RTMP format.

**Comment by dnewman-gpsw** [comment]
Unfortunately this is beyond Labs development scope.  Labs controls and manipulates existing camera modes and functions, there is no SRT format to enable in camera.

**Reply by Blackwolf8202** [reply]
Thanks for the reply. But who is in charge of these additions to the gopro then? Is it something that can be expressed as wishes?

**Reply by dnewman-gpsw** [reply]
I've seen requests like this on the GoPro community pages, there are folks that document those.
